libraries/Servo/src/nrf52/Servo.cpp

```diff
@@ -47,53 +47,58 @@ uint8_t Servo::attach(int pin, int min, int max)
     return INVALID_SERVO;
   }
   bool succeeded = false;
-  pinMode(pin, OUTPUT);                                   // set servo pin to output
-  servos[this->servoIndex].Pin.nbr = pin;
 
-  if (min < MIN_PULSE_WIDTH) {
-    min = MIN_PULSE_WIDTH;
-  }
-  if (max > MAX_PULSE_WIDTH) {
-    max = MAX_PULSE_WIDTH;
-  }
+  if (min < MIN_PULSE_WIDTH) min = MIN_PULSE_WIDTH;
```

Comment:
The lack of braces is a common cause of bugs.  therefore, I recommend against removing them.   Even if you place them on the same line, I recommend keeping the curly braces.

Reply:
OK, this actually make sense, but it is not eye candy. Though I will change it.